### DISCUSSION TITLE
[Chrome插件] Octotree: GitHub Code Tree

### DISCUSSION THREAD

**Discussion by tyn1998** [X-lab2017/open-wonderland#125] [1️⃣ 新鲜事]
插件地址：https://chrome.google.com/webstore/detail/octotree-github-code-tree/bkhaagjahfmjljalopjnoealnfndnagc

实用又美观：

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/32434520/178184027-80d89f94-10da-408e-ab67-6133a10cbfda.png">

免费版够用了，但也有付费方案：

<img width="917" alt="image" src="https://user-images.githubusercontent.com/32434520/178184099-248277bd-eb69-44a9-a5c4-9f21f5e5d918.png">

**Comment by will-ww** [comment]
棒呀！有效果的付费方案，我们也是考虑的~

**Comment by xgdyp** [comment]
github.dev我觉得也挺好用的，现在找代码都直接按.进dev里找

**Reply by tyn1998** [reply]
这几天修crx的bug，需要看别的项目的源码，已经体验好几次了，赞不绝口！

**Comment by TieWay59** [comment]
一直在用
